tensorflow/python/framework/ops.py

```diff
@@ -686,6 +686,25 @@ def __copy__(self):
   # with ndarrays.
   __array_priority__ = 100
 
+  def __array__(self):
+    try:
+      name = self.name
+    except ValueError:
```

Comment:
This should not be necessary, all symbolic Tensors are expected to have a name, AFAIK. Are you seeing test failures without it?